.travis-ocaml.sh

```diff
@@ -256,7 +256,7 @@ case $TRAVIS_OS_NAME in
     linux) install_on_linux ;;
 esac
 
-OPAM_SWITCH=${OPAM_SWITCH:-$OCAML_FULL_VERSION}
+OPAM_SWITCH=${OPAM_SWITCH:-ocaml-base-compiler.$OCAML_FULL_VERSION}
```

Comment:
I guess because they define `OPAM_SWITCH`?  I launched the failed build and it ran fine with that modification — but I must admit that I haven't investigated deeply the scripts.

Reply:
Ha yes, indeed :-) Thanks for the fix